src/lib/elastic.js

```diff
@@ -2,6 +2,23 @@ const path = require('path')
 const _ = require('lodash')
 const fs = require('fs');
 const jsonFile = require('jsonfile')
+const es = require('elasticsearch')
+
+function getClient(config) {
```

Comment:
Why are we creating a new client every time? 

Why are we not caching the client and return that when we ask for a client. I don't know if that changes something for performance or if it even works with the ES client. 

But this would be something we could now do when it works. So could you maybe test it?

Reply:
I think that caching the client may be problematic (it would be kind of in-module caching, not sure if it won't interfere between user requests - for example the resulting cursors): I think the downside on performance is close to zero as Elastic is doing mostly HTTP requests